### ISSUE TITLE
Doesn't work in USI

### ISSUE THREAD

**Issue by Bluscream** [StylishThemes/GitHub-Dark-Script#26]
Browser: [Mozilla Firefox Aurora](https://play.google.com/store/apps/details?id=org.mozilla.fennec_aurora&hl=en&pcampaignid=APPU_1_DbFAWLzWD8GbsgGS0KuYBg) 52.0a2
Userscript Framework: [Unified Script Injector for Firefox](https://addons.mozilla.org/en-US/android/addon/userunified-script-injector/) 0.4.4-b

![tmp_3313-screenshot_20161202-001721-1914132812](https://cloud.githubusercontent.com/assets/3318223/20816929/2539d1dc-b825-11e6-8058-3c185e84d2b1.png)

**Comment by Bluscream** [comment]
This is the internal FAQ page if it helps:
![tmp_3313-screenshot_20161202-0026111763340072](https://cloud.githubusercontent.com/assets/3318223/20817101/071bc42a-b826-11e6-8e3d-30e764471af8.jpg)

**Comment by Mottie** [comment]
Hi @Bluscream!

This sounds like an issue with USI... if we switch the `@run-at` to anything but `document-start`, you'll see a flash of white because the style won't get applied until after the document body has been created.

I would have to say that this would be a "will not fix" issue. Wait until USI is out of alpha, then we'll see what happens.

Correction: The addon is in beta... the note at the top still says alpha; but still it's not 100% compatible with Greasemonkey.

**Comment by Bluscream** [comment]
Yeah i didn't expeted this to be a easy fix. I just wanted to let you know about it :)
